epochStart/mock/accountsAdapterStub.go

```diff
@@ -2,7 +2,6 @@ package mock
 
 import (
 	"context"
-	"errors"
```

Comment:
does the epochStart/mock/accountsAdapter.go is an empty file?

Reply:
`accountsAdapterStub` is not empty.